### PR TITLE
fix: Downgrade to 10.0.22621 for Build Tools compatibility

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): related to https://github.com/unoplatform/uno/issues/18840

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

The latest version of Build Tools is unsupported by Microsoft Store and block users from publishing their apps

## What is the new behavior?

Downgraded until support is added

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Sdk/packages.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
   },
   {
     "group": "WinAppSdkBuildTools",
-    "version": "10.0.26100.1742",
+    "version": "10.0.22621.3233", // Keep this version until https://github.com/microsoft/WindowsAppSDK/issues/4480 is resolved
```
Comment (jeromelaban) [inline src/Uno.Sdk/packages.json:124]: It's likely that the automated tooling will force the upgrade to the latest version.  

We probably should be excluding this group here:

https://github.com/unoplatform/uno.templates/blob/6974d53309e8bb2de1ce13d6f23c0d5149d10020/tools/Uno.Sdk.Updater/Program.cs#L294-L298

Comment (MartinZikmund) [reply src/Uno.Sdk/packages.json]: Done

**Comment by nickrandolph** [comment]
@MartinZikmund we should adjust the unoapp template for similar reason

**Review by jeromelaban — APPROVED** [review]

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**